docs/config_reference.rst

```diff
@@ -398,6 +398,7 @@ System Partition Configuration
    Available values are the following:
 
    - ``alps``: Parallel programs will be launched using the `Cray ALPS <https://pubs.cray.com/content/S-2393/CLE%205.2.UP03/cle-xc-system-administration-guide-s-2393-5203-xc/the-aprun-client>`__ ``aprun`` command.
+   - ``clush``: Parallel programs will be launched using the `clush` command. This launcher uses the partition's :attr:`~config.systems.partitions.access` property in order to determine the options to be passed to `clush`.
```

Comment:
Perhaps we should also add a link to the `clush` main page.

Reply:
I'll add that. Any other thoughts on things you'd like added?